src/complexity.cc

```diff
@@ -193,63 +201,50 @@ std::vector<BenchmarkReporter::Run> ComputeStats(
     CHECK_EQ(reports[0].benchmark_name, run.benchmark_name);
     CHECK_EQ(run_iterations, run.iterations);
     if (run.error_occurred) continue;
-    real_accumulated_time_stat +=
-        Stat1_d(run.real_accumulated_time / run.iterations);
-    cpu_accumulated_time_stat +=
-        Stat1_d(run.cpu_accumulated_time / run.iterations);
-    items_per_second_stat += Stat1_d(run.items_per_second);
-    bytes_per_second_stat += Stat1_d(run.bytes_per_second);
+    real_accumulated_time_stat.emplace_back(run.real_accumulated_time);
+    cpu_accumulated_time_stat.emplace_back(run.cpu_accumulated_time);
+    items_per_second_stat.emplace_back(run.items_per_second);
+    bytes_per_second_stat.emplace_back(run.bytes_per_second);
     // user counters
     for(auto const& cnt : run.counters) {
       auto it = counter_stats.find(cnt.first);
       CHECK_NE(it, counter_stats.end());
-      it->second.s += Stat1_d(cnt.second);
+      it->second.s.emplace_back(cnt.second);
     }
   }
 
-  // Get the data from the accumulator to BenchmarkReporter::Run's.
-  Run mean_data;
-  mean_data.benchmark_name = reports[0].benchmark_name + "_mean";
-  mean_data.iterations = run_iterations;
-  mean_data.real_accumulated_time =
-      real_accumulated_time_stat.Mean() * run_iterations;
```

Comment:
We've had attempts in the past to clean up this `run_iterations` thing as it does seem like it isn't necessary. However, and I wish i could remember or dig up the reasoning, it ends up coming in to play for some combinations of runs/threads/repetitions.

However, if your change shows expected results when we run the tests we have, i'm comfortable with the notion of the change.